src/component/sidebar/sidebar.tsx

```diff
@@ -16,26 +18,35 @@ import './i18n/index'
 // eslint-disable-next-line max-lines-per-function
 export const Sidebar: FC = () => {
   const { t } = useTranslation('sidebar')
-  const theme = useAppStore(store => store.theme)
-  const isSidebarExpanded = useAppStore(store => store.isSidebarExpanded)
-  const collapseSidebar = useAppStore(store => store.collapseSidebar)
-  const expandSidebar = useAppStore(store => store.expandSidebar)
-  const switchTheme = useAppStore(store => store.switchTheme)
+  const { collapseSidebar, expandSidebar, isSidebarExpanded, switchTheme, theme } = useAppStore(
+    state => ({
+      collapseSidebar: state.collapseSidebar,
+      expandSidebar: state.expandSidebar,
```

Comment:
Just like you have a function to change the theme, you could have a function to switch the state of the sidebar (expanded/collapsed). This will simplify your code a bit.

Reply:
Yes it's a good idea. 
It's just that sometimes it's only the `collapse` or `expand` function that is used, like when we are clicking on the close icon.
So I'm not sure that we should make the changes and use a general `toggle` function... 
It will work of course but, the intention is not to toggle but juste to close the sidebar for example...
 What do you think @ccamel ?

Reply:
As we have seen together, the idea is to offer the client all the possibilities: `collapseSidebar`, `expandSidebar`, `toggleSidebar`, in order to facilitate his consumption.